### PR TITLE
[general] Minor fixes for extending django-ipam

### PR DESCRIPTION
**Pull request by anurag-ks**
Small changes which were required while extending `django-ipam` into `openwisp-ipam`.

### CODE REVIEW

File: django_ipam/models.py [hunk 1]
```diff
@@ -7,12 +7,14 @@ class Subnet(AbstractSubnet):
     class Meta(AbstractSubnet.Meta):
         abstract = False
         swappable = swappable_setting('django_ipam', 'Subnet')
+        app_label = 'django_ipam'
```
Comment (nemesifier) [inline django_ipam/models.py]: why is this necessary? @rohithasrk is not doing this in `django-freeradius`

File: django_ipam/models.py [hunk 2]
```diff
@@ -7,12 +7,14 @@ class Subnet(AbstractSubnet):
...
 
 
 class IpAddress(AbstractIpAddress):
     class Meta(AbstractIpAddress.Meta):
         abstract = False
         swappable = swappable_setting('django_ipam', 'IpAddress')
+        app_label = 'django_ipam'
```
Comment (nemesifier) [inline django_ipam/models.py]: why is this necessary? @rohithasrk is not doing this in `django-freeradius`

**Review by nemesifier — COMMENTED** [review]
looks good but I have a doubt regarding `app_label` in the model `Meta`, could you clarify?

**Comment by anurag-ks** [comment]
@nemesisdesign thankyou for the review, I had to add the the `app_label` because of this error - 
`RuntimeError: Model class django_ipam.models.Subnet doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.`, 
I will try to look for correct solution for this.

**Comment by nemesifier** [comment]
@anurag-ks take a look at what Rohith is doing, ask him questions if needed

**Comment by anurag-ks** [comment]
@nemesisdesign another thing I noticed is that when I add the OrgMixin or rather the organization field to the models, the test cases under `openwisp-ipam` fail. Since the `organization` field cannot be empty.

**Review by nemesifier — APPROVED** [review]
:+1: glad to see the patch has been simplified a lot, this is the way to go
